lib/cli/output/init.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Please explore the output formatting packages using in the autolabcli application and use the necessary ones. Once you select a few packages, please update the [notes on libraries](https://github.com/AutolabJS/autolabcli/wiki/Notes-on-Libraries) wiki page.